app/views/smart_proxies/plugins/puppet/_puppet_ca.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="row">
+  <div class="col-md-12">
+    <div class="row">
+      <h4><%= _('Puppet CA') %></h4>
+    </div>
+    <%= render :partial => 'smart_proxies/plugins/plugin_version', :locals => {:feature => 'puppetca', :proxy => @smart_proxy } %>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Hosts count') %>
+      </div>
+      <div class="col-md-8">
+        <%= link_to(@smart_proxy.puppet_ca_hosts.count, hosts_path(:search => "puppet_ca = #{@smart_proxy.name}")) %>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Valid certificates') %>
+      </div>
+      <div class="col-md-8">
+        <% valid =  SmartProxies::PuppetCA.find_by_state(@smart_proxy, "valid").count %>
+        <%= display_link_if_authorized(valid, hash_for_smart_proxy_puppetca_index_path(:smart_proxy_id => @smart_proxy, :state => 'valid').
+                                                          merge(:auth_object => @smart_proxy, :permission => 'view_smart_proxies_puppetca', :authorizer => authorizer)) %>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Pending certificates') %>
+      </div>
+      <div class="col-md-8">
+        <% pending = SmartProxies::PuppetCA.find_by_state(@smart_proxy, "pending").count %>
+        <%= display_link_if_authorized(pending, hash_for_smart_proxy_puppetca_index_path(:smart_proxy_id => @smart_proxy, :state => 'pending').
+                                              merge(:auth_object => @smart_proxy, :permission => 'view_smart_proxies_puppetca', :authorizer => authorizer)) %>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('Autosign entries') %>
+      </div>
+      <div class="col-md-8">
+        <% autosign = ProxyAPI::Puppetca.new(:url => @smart_proxy.url).autosign.count %>
+        <%= display_link_if_authorized(autosign, hash_for_smart_proxy_autosign_index_path(:smart_proxy_id => @smart_proxy).
```

Comment:
since this entire tab does not load over ajax, if there is error here the entire show page wont render.